patrol/cli.py

```diff
@@ -20,10 +20,32 @@ def events(organization, project_name):
         click.echo('{eventID}: {message}'.format(**event))
 
 
+@cli.command(help='Lists project events')
+@click.argument('organization')
+@click.argument('project_name')
+@click.argument('event_id')
+def event(organization, project_name, event_id):
+    click.secho('event {} for {}'.format(event_id, project_name))
+    event = patrol.event(organization, project_name, event_id)
+    click.echo('{eventID}: {message}'.format(**event))
+
+
 @cli.command(help='Lists project issues')
 @click.argument('organization')
 @click.argument('project_name')
 def issues(organization, project_name):
     click.secho('issues for {}'.format(project_name))
     for issue in patrol.issues(organization, project_name):
         click.echo('{id}: {title}'.format(**issue))
+
+
+@cli.command(help='Retrive issue')
```

Comment:
I believe it's better to keep it as two commands.. less error-prone usage I believe, what do you think?

Reply:
Hm Okay! I agree with that, I just thought it would look better like this. I'll put it back as two commands.